eng/native/build-commons.sh

```diff
@@ -48,7 +48,9 @@ check_prereqs()
 
 build_native()
 {
-    eval "$__RepoRootDir/eng/native/version/copy_version_files.sh"
+    if [[ ! -e "$__RepoRootDir/artifacts/obj/_version.c" ]]; then
+        eval "$__RepoRootDir/eng/native/version/copy_version_files.sh"
+    fi
```

Comment:
@jkoritzinsky, in official build, we always build from clean repo so I assumed this won't be a problem.

Reply:
Yes we always build from a clean clone for an official build.